src/aims_ui/page_error.py

```diff
@@ -7,10 +7,15 @@ def page_error(
     page_name,
 ):
 
+  if api_response.json().get('errors',[''])[0].get('message',None) != None:
```

Comment:
dict.get provides None as a default, so all of the `.get('...', None)` calls here can be turned into `.get('...')`.

Reply:
Also, if you're trying to search for errors, couldn't you use the length of the `errors` array instead, ala `if len(api_response.json().get('errors', [])) > 0`?

Reply:
Changed to len(api_response.json().get('errors', [])) > 0